jumpscale/packages/farmmanagement/actors/farm_management.py

```diff
@@ -2,6 +2,8 @@
 from jumpscale.clients.explorer.models import Farm
 from jumpscale.loader import j
 
+import requests
```

Comment:
unused import

Reply:
> unused import

used to catch the HTTPError exception in line 22